src/commons/big_query/copy_job_async/copy_job/copy_job_service.py

```diff
@@ -54,18 +55,21 @@ def __schedule(source_big_query_table, target_big_query_table, job_id,
             }
         }
         try:
-            job_reference = BigQuery().insert_job(
-                target_big_query_table.get_project_id(), job_data)
+            job_reference = BigQuery().insert_job(target_project_id, job_data)
             logging.info("Successfully insert: %s", job_reference)
             return job_reference
         except HttpError as bq_error:
             copy_job_error = BigQueryJobError(bq_error,
                                               source_big_query_table,
                                               target_big_query_table)
-            if copy_job_error.should_be_retried():
+            if copy_job_error.is_deadline_exceeded():
+                job_json = CopyJobService.__get_job(job_id, target_project_id,
```

Comment:
What will happen when you'll get 404 here?

Reply:
Discussed offline